app/src/main/java/edu/drexel/lapcounter/lapcounter/frontend/CurrentWorkoutActivity.java

```diff
@@ -219,44 +251,34 @@ protected void onStart() {
             return;
```

Comment:
AHA! This is the offending line. the first time this happens, mDeviceAddress is null since there was no connected device